config/operator.yaml

```diff
@@ -25,12 +25,17 @@ spec:
                 fieldRef:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.name
+              value: "knative-serving-operator"
```

Comment:
aligned the `OPERATOR_NAME` with how it is done for `serving-operator` ....

Reply:
Shouldn't it be knative-eventing-operator?

Reply:
@houshengbo thanks - I just corrected that !